gramps/cli/clidbman.py

```diff
@@ -113,7 +115,7 @@ class CLIDbManager:
     ICON_LOCK = 2
     ICON_OPEN = 3
 
-    ICON_MAP = {
+    ICON_MAP: Dict[int, Optional[str]] = {
```

Comment:
I think you can use `from __future__ import annotations` and then use the simpler:
```suggestion
    ICON_MAP: dict[int, str | None] = {
```
so long as `mypy` is new enough. And then apply as necessary to most of the `typing` imports in other files.

Reply:
That's correct but eventually it's a style decision as long as we have to support Python 3.8 or earlier. When we drop 3.8 support, we can switch to the new syntax without needed the future import.

Reply:
True, it's a matter of style, but it's a lot easier to have it go in the new way now, and review that once, plus a simple removal of future imports, rather than review this now plus have to do some cleanup that touches everything all over again at some point in the future.

Reply:
I don't mind changing it. Perhaps we will anyway require Python 3.9 for Gramps 5.3? Python 3.8 EOL is in 5 months from now.

Reply:
Changed everywhere now!